### PR TITLE
Soko3

### PR DESCRIPTION
**Pull request by solongoch**
1. added logout button as a menu in dropdown of Profile picture in Navbar and after logout, redirect to the landing page
2. show real avatar in Navbar
3. added "NotFound" page
4. created SearchProfile component and search component logic
5. created ProfilePage comp, separated ProfileHeader as a component
6. getting real data from db in ProfileHeader
7.  changed profile routes into `/profile/:username` in App.js

UPDATE:
1. getting datas in ProfilePage, passing the datas to ProfileHeader and ProfilePosts
2. improved search logic, added PROFILE_LOADING reducer

### CODE REVIEW

File: client/src/components/Layout/SearchProfile.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  render() {
+    return (
+      <div className="search-div col-lg-4 col-md-4 col-sm-4">
+        <form className="form-inline" onSubmit={this.onSubmit}>
+          {/* <i className="fas fa-search" aria-hidden="true"></i> */}
+          <input
+            id="search"
+            type="search"
+            name="search"
```
Comment (slpriya) [inline client/src/components/Layout/SearchProfile.js:41]: value attribute  is missing

**Review by slpriya — APPROVED** [review]
value attribute is missing in SearchProfile.js

**Review by Emily484 — APPROVED** [review]
looks good

**Review by slpriya — APPROVED** [review]
Looks good

**Pull request merged**